src/clj/ataru/tarjonta_service/tarjonta_service.clj

```diff
@@ -1,15 +1,19 @@
 (ns ataru.tarjonta-service.tarjonta-service
   (:require
-    [ataru.tarjonta-service.tarjonta-client :as client]))
+    [ataru.tarjonta-service.tarjonta-client :as client]
+    [ataru.virkailija.user.organization-client :refer [oph-organization]]))
 
 (defn get-forms-in-use
-  []
-  (reduce (fn [acc1 {:keys [avain haut]}]
-            (assoc acc1 avain
-                        (reduce (fn [acc2 haku]
-                                  (assoc acc2 (:oid haku)
-                                              {:haku-oid  (:oid haku)
-                                               :haku-name (get-in haku [:nimi :kieli_fi])}))
-                                {} haut)))
-          {}
-          (client/get-forms-in-use)))
\ No newline at end of file
+  [organization-service username]
+  (let [direct-organization-oids (map :oid (.get-direct-organizations organization-service username))
+        all-organization-oids    (map :oid (.get-all-organizations organization-service direct-organization-oids))
```

Comment:
without knowing anything about organization service, is this slow or fast to get all organizations?